### DISCUSSION TITLE
exceptions:UnavailableException

### DISCUSSION THREAD

**Discussion by zjafarian** [sceneview/sceneform-android#64] [Q&A]
i get this Erorr 'com.google.ar.core.exceptions.UnavailableException' when run every samples.
i use Android studio 4.1.2

**Comment by ThomasGorisse** [comment]
Can you send the full stack trace ?

**Comment by zjafarian** [comment]
Thanks for the reply, I found the problem. The problem was Emulator

**Comment by ThomasGorisse** [comment]
You can check the emulator known working configuration here: [Emulator - Known working configuration](https://thomasgorisse.github.io/sceneform-android-sdk/emulator)

**Reply by zjafarian** [reply]
Thanks for link :)

**Comment by zjafarian** [comment]
sorry, I have a problem with run 'sample-augmented-image', after the run, this error


![Screenshot from 2021-05-17 11-09-45](https://user-images.githubusercontent.com/68461369/118442895-7114be80-b700-11eb-9eaf-19db4616af7f.png)

**Comment by ThomasGorisse** [comment]
Please check your emulator/computer or device OpenGL version. And go to the advanced emulator settings

![](https://user-images.githubusercontent.com/6597529/117982203-0fd2a100-b336-11eb-96b0-35e5e18fec5e.png)

Then reboot your emulator
